packages/layout/src/components/beta/beta.tsx

```diff
@@ -30,14 +32,16 @@ export const BetaHeader: React.FC<BetaHeaderProps> = ({ text, mail }) => {
 						</Link>{' '}
 						will help us improve it.
 					</>
-			  );
+				)}
+			</Span>
+		);
 	}, [text, mail]);
 
 	return (
 		<DocWidth className={style.beta}>
 			<AppWidth>
 				<Flex cfg={{ alignItems: 'center', px: 6 }}>
-					<P
+					<Span
```

Comment:
`<p>` is the right element here, from an accessibility point of view. It can be styled as needed so there's no reason to change it to `<span>`.

Reply:
I've put the <P> back instead of the <Span>